view/adminhtml/templates/menu/nodes.phtml

```diff
@@ -33,21 +33,27 @@
                     <?= __($formAttribute->getLabel()); ?>
                 </label>
                 <div class="admin__field-control control">
-                <?php if ($formAttribute->getType() != 'wysiwyg'): ?>
-                    <input class="input-text admin__control-text"
-                           data-mage-init='{"snowMenuEditorRename":{"type":"<?= $formAttribute->getCode(); ?>"}}'
-                           id="snowmenu_<?= $formAttribute->getId(); ?>"
-                           type="<?= $formAttribute->getType(); ?>"
-                    />
-                <?php else: ?>
-                <?php
-                    $wysiwyg = $block->getEditor('snowmenu_' . $formAttribute->getId(), $formAttribute->getId());
-                    $wysiwyg->setDataMageInit(
-                        "{\"snowMenuEditorRename\":{\"type\":\"" . $formAttribute->getCode() . "\"}}"
-                    );
-                    echo $wysiwyg->getElementHtml();
-                ?>
-                <?php endif; ?>
+                    <?php if ($formAttribute->getType() == 'wysiwyg'): ?>
+                        <?php
+                        $wysiwyg = $block->getEditor('snowmenu_' . $formAttribute->getId(), $formAttribute->getId());
+                        $wysiwyg->setDataMageInit(
+                            "{\"snowMenuEditorRename\":{\"type\":\"" . $formAttribute->getCode() . "\"}}"
```

Comment:
Double quotes for php code are not needed. You can use single quotes to avoid adding slashes.

Reply:
done